h/static/scripts/test/app-controller-test.coffee

```diff
@@ -168,6 +168,28 @@ describe 'AppController', ->
     $scope.$broadcast(events.USER_CHANGED, {initialLoad: false})
     assert.calledOnce(fakeRoute.reload)
 
+  describe 'on "beforeAnnotationCreated"', ->
+    it 'calls $scope.clearSelection()', ->
+      createController()
+      sandbox.spy($scope, 'clearSelection')
+
+      $rootScope.$emit('beforeAnnotationCreated', {})
+
+      assert(
+        $scope.clearSelection.called
+        'It should clear any selection that exists in the sidebar before '
```

Comment:
Could you move this text into a comment and do `assert.called($scope.clearSelection)`. That'll be easier to read if the test fails and the detail will still be available in the test function for debugging.